src/journeys_utils.js

```diff
@@ -266,12 +266,44 @@ journeys_utils.addHtmlToIframe = function(iframe, html, userAgent) {
 	else {
 		bodyClass = 'branch-banner-desktop';
 	}
-
 	var iframedoc = iframe.contentDocument || iframe.contentWindow.document;
 	iframedoc.head = iframedoc.createElement('head');
 	iframedoc.body = iframedoc.createElement('body');
 	iframedoc.body.innerHTML = html;
 	iframedoc.body.className = bodyClass;
+	var metaTag = iframedoc.querySelector('meta[name="accessibility"]');
+	if (metaTag && metaTag.content === 'wcag') {
+		var scriptTag = iframedoc.createElement('script');
+		scriptTag.type = 'text/javascript';
+		scriptTag.text = `
+			const  focusableElements =
```

Comment:
small/minor change : change const to var. Our closure compiler is very old and We have not yet tested ES6 features fully.So till the time we upgrade our closure compiler and fully test ES6 support, better to use var.

Reply:
This is a very good point, I've updated the code. Please take a look again, thanks!